sympy/stats/crv_types.py

```diff
@@ -1655,6 +1683,13 @@ def Kumaraswamy(name, a, b):
 class LaplaceDistribution(SingleContinuousDistribution):
     _argnames = ('mu', 'b')
 
+    set = Interval(-oo, oo)
+
+    @staticmethod
+    def check(mu, b):
+        _value_check(mu.is_real, "mu, location must be real.")
```

Comment:
I would suggest you be consistent for such cases. For once, I was confused by the error message and maybe end users might face similar problems.